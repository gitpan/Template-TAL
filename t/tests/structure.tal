<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    <div tal:replace="html" />
    <div tal:replace="structure html" />
  </body>
</html>
