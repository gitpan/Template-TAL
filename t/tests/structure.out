<html xmlns:tal="http://xml.zope.org/namespaces/tal">
  <body>
    &lt;p&gt;this is html&lt;/p&gt;
    <p>this is html</p>
  </body>
</html>
